index.js

```diff
@@ -0,0 +1,59 @@
+let form = document.getElementById("new-task-form");
+let input = document.getElementById("new-task-input");
+let message = document.getElementById("message");
+let posts = document.getElementById("tasks");
+
+form.addEventListener("submit", (e) => {
+  e.preventDefault();
+  formValidation();
+});
+let formValidation = () => {
+  if (input.value === "") {
+    message.innerHTML = "Input field can not be blank...";
+  } else {
+    message.innerHTML = "";
+    acceptData();
+  }
+};
+let data = [];
+
+let acceptData = () => {
+  data.push({
+    text: input.value,
+  });
+  localStorage.setItem("data", JSON.stringify(data));
+  createPost();
+};
+let createPost = () => {
+  posts.innerHTML = "";
+  data.map((x, y) => {
```

Comment:
'x' is data, 'y' is unnecessary, i will remove it

Reply:
Well, 'y' was my id counter, but app works fine without it

Reply:
Okay, you should name your x,y differently because x and y can be anything, right? So try naming them (task, index) and don't remove your id , whenever you use  .map() to render some elements you need to assign each element unique id.